### PR TITLE
Basic News and Healthy thinking importer added

### LINKED ISSUE #7: News and Press Release Pages Importer
Refer [sheet](https://adobe.sharepoint.com/:x:/r/sites/HelixProjects/_layouts/15/Doc.aspx?sourcedoc=%7B15EAEB2A-C831-4779-922B-6EC0F4E36CD6%7D&file=sunstar_com-crawl-report.xlsx&action=default&mobileredirect=true)

### PR DESCRIPTION
**Pull request by jindaliiita**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #7 

Test URLs:
- Before: https://main--sunstar--hlxsites.hlx.page/
- After: https://news-importer--sunstar--hlxsites.hlx.page/

- [ ]  If you are adding a new block or a variation to an existing block, has it been added in the [block-library](https://adobe.sharepoint.com/:x:/r/sites/HelixProjects/Shared%20Documents/sites/sunstar/sunstar/sidekick/library.xlsx?d=w7f8a415fe192444caa87806b565bb62d&csf=1&web=1&e=oJWCS0) ?

### CODE REVIEW

File: tools/importer/import-news-press-healthythinking.js [hunk 1]
```diff
@@ -0,0 +1,373 @@
...
+* @param {HTMLDocument} document The document
+*/
+const addCardsBlockFragmentFromSection = (document) => {
+  document.querySelectorAll('div.section-container').forEach((section) => {
+    const block = [['Fragment']];
+    const newsPressCard = section.parentElement.className.includes('news-featured');
+
+    if (newsPressCard) {
+      const a = document.createElement('a');
+      a.href = 'https://main--sunstar--hlxsites.hlx.page/_drafts/piyush/title'; // todo piyush change this before merging
```
Comment (JiangLong2019) [inline tools/importer/import-news-press-healthythinking.js:154]: TODO needs to do? Haha

Comment (jindaliiita) [reply tools/importer/import-news-press-healthythinking.js]: 😃

**Comment by jindaliiita** [comment]
@sdmcraft @JiangLong2019 This PR is ready for review. Basic importer for news and healthy thinking page.
Kindly review this.

**Review by sdmcraft — COMMENTED** [review]
Looks good. Please consider contributing common functionality which could be useful across specific importers to https://github.com/hlxsites/sunstar/blob/main/tools/importer/utils.js . For e.g. `createMetadata` , `extractEmbed` etc. Also you may find something already there which you could reuse here.

**Review by sdmcraft — APPROVED** [review]

**Pull request merged**